source/settings/dovecot_core_settings.rst

```diff
@@ -1460,6 +1460,10 @@ There are currently defaults for the following fields:
 * ``support-url``
 * ``support-email``
 
+.. versionadded:: 2.3.10
+
+* ``revision``
```

Comment:
I previously changed a bunch of versionadded to be indented because it wasn't clear if it meant the above or below items. This can probably be:

```
* ``revision``

   .. versionadded:: 2.3.10
```

Reply:
I intentionally added it above so it is clear only revision is added in 2.3.10

![image](https://user-images.githubusercontent.com/4457135/72257107-7a28e380-3613-11ea-8bb7-1abef5090316.png)
 vs

![image](https://user-images.githubusercontent.com/4457135/72257163-9e84c000-3613-11ea-99b2-a423b6b4bf78.png)

Reply:
But with the indentation it looks like:

<img width="658" alt="Screen Shot 2020-01-13 at 14 55 06" src="https://user-images.githubusercontent.com/1644825/72257613-bd378680-3614-11ea-8788-56b258256391.png">

Reply:
I checked the doc and found that versionadded takes an optional second argument that can be used to describe the change. Moved versionadded to after the list with a short description.